src/main/webapp/components/BuildSnapshotContainer.js

```diff
@@ -9,7 +9,15 @@ import {BuildSnapshot} from "./BuildSnapshot";
  */
 export const BuildSnapshotContainer = React.memo((props) =>
   {
-    const SOURCE = '/builders';
+    // Number of entries to be pulled.
+    const NUMBER = 2;
```

Comment:
why is NUMBER 2? I'd expect something like a dozen or more entries on each page?
Could you clarify if this is an intermediate PR just to get the initial integration working and subject to change later?

Reply:
2 here is used to get the initial integration working. It will be subject to change later, following a discussion on an appropriate value to use.

Reply:
consider adding a TODO here that having constants about this here is not the final code state